### ISSUE TITLE
Zoom fails to load on ubuntu 22.04 since update to 5.13.5.431

### ISSUE THREAD

**Issue by zebardy** [flathub/us.zoom.Zoom#369]
SInce the update installed ~5 hours ago zoom no longer starts. Bellow is the output of trying to run zoom from the terminal. 

``` $ flatpak run -v us.zoom.Zoom --verbose
F: No installations directory in /etc/flatpak/installations.d. Skipping
F: Opening system flatpak installation at path /var/lib/flatpak
F: Opening user flatpak installation at path /home/aaron/.local/share/flatpak
F: Opening user flatpak installation at path /home/aaron/.local/share/flatpak
F: Opening system flatpak installation at path /var/lib/flatpak
F: Skipping parental controls check for app/us.zoom.Zoom/x86_64/stable since parental controls are disabled globally
F: Opening user flatpak installation at path /home/aaron/.local/share/flatpak
F: Opening system flatpak installation at path /var/lib/flatpak
F: /var/lib/flatpak/runtime/org.freedesktop.Platform/x86_64/22.08/3616ca621531deb9904cb2ebe436211609d503e481122f57a50067ecb9979e63/files/lib32 does not exist
F: Cleaning up unused container id 2941584184
F: Cleaning up per-app-ID state for us.zoom.Zoom
F: Allocated instance id 2569807373
F: Add defaults in dir /us/zoom/Zoom/
F: Add locks in dir /us/zoom/Zoom/
F: Allowing wayland access
F: Allowing x11 access
F: Allowing pulseaudio access
F: Running 'bwrap --args 40 xdg-dbus-proxy --args=43'
F: Running 'bwrap --args 40 zoom --verbose'
```

After outputting this execution stops and not zoom windows appear on the screen.

**Comment by pjohno** [comment]
This is happening to me as well. Running from the command line I get no errors or logs. I've tried reinstalling and deleting data but nothing works. I'm downgrading to 5.13.4.711 for the time being.

**Comment by ebousse** [comment]
Same problem with Fedora 37 running GNOME here.

**Comment by bogdanvlviv** [comment]
There are some important security fixes in 5.13.5.431 - using 5.13.4.711 is risky. How can we get this version working? Is this flatpak's issue or Zoom 5.13.5.431 itself?
